src/rdkafka.h

```diff
@@ -3088,6 +3088,24 @@ rd_kafka_offsets_for_times (rd_kafka_t *rk,
                             int timeout_ms);
 
 
+/**
+ * @brief Allocate memory using the librdkafka internal allocator
```

Comment:
"using the same allocator librdkafka uses." is more correct, it currently implies that there's an internal librdkafka allocator which there is not.